code/go/0chain.net/smartcontract/storagesc/config_settigns.go

```diff
@@ -295,6 +297,7 @@ var (
 		"cost.challenge_response":          {CostChallengeResponse, smartcontract.Cost},
 		"cost.generate_challenges":         {CostGenerateChallenges, smartcontract.Cost},
 		"cost.add_validator":               {CostAddValidator, smartcontract.Cost},
+		"cost.update_validator_settings":   {CostUpdateValidatorSettings, smartcontract.Cost},
```

Comment:
Update sc.yaml to add the costs in `storagesc.cost` field

Reply:
use 100 as default

Reply:
Done